docs/scenarios/library-authoring/trimming.md

```diff
@@ -0,0 +1,177 @@
+# How to trim your package dependencies
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers what you need to know about "trimming" your package dependencies to use only the packages you need for .NET Core RC2.
+
+## What trimming means
+
+.NET Core RC2 introduces the .NET Standard Library metapackage, which is a NuGet package composed other packages.  It is curated to be appropriate for developing libraries.  However, there's a good chance that your library won't use every single package it contains.  When authoring a library and distributing it over NuGet, it's a best practices to "trim" your dependencies down to only the packages you actually use.  This results in a smaller overall footprint for NuGet packages.
+
+## How to trim
+
+Currently, there is no official tooling which supports the trimming of packages.  You'll have to do it manually.  The general process will look like this:
+
+1. Reference `NETStandard.Library` version `1.5.0-rc2`.
+2. Restore packages with `dotnet restore` from the command line.
+3. Inspect the `project.lock.json` file and find the `NETSTandard.Library` section.  It's near the beginning of the file.
+4. Copy all of the listed packages under `dependencies`.
+5. Remove the `.NETStandard.Library` reference and replace it with the copied packages.
+6. Remove references to packages you don't need.
+
+You can find out which packages you don't need by one of the following ways:
+
+1. Trial and error.  This involves removing a package, restoring, compiling, and seeing if your library still compiles.
+2. Using a tool such as [ILSpy](ilspy) or [.NET Reflector](reflector) to peek at references to see what your code is actually using.  You can then remove packages which don't correspond to types you're using.
```

Comment:
These links don't seem to work. docfx says:

> Warning: \[Build Document](docs/scenarios/library-authoring/trimming.md)Invalid file link(~/docs/scenarios/library-authoring/ilspy)
Warning: \[Build Document](docs/scenarios/library-authoring/trimming.md)Invalid file link(~/docs/scenarios/library-authoring/reflector)